### PR TITLE
add mbutil

### PR DESCRIPTION
**Pull request by rluria14**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@oblute @benhuff will you be maintainers?

**Comment by oblute** [comment]
@rluria14 yes, I agree to be a co-maintainer

**Comment by benhuff** [comment]
@rluria14 yes, I will be a maintainer.

File: recipes/mbutil/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - mbutil
+
+about:
+  home: https://github.com/mapbox/mbutil
+  license: BSD
```
Comment (marcelotrevisani) [inline recipes/mbutil/meta.yaml]:
```suggestion
  license: BSD 3-Clause
```

File: recipes/mbutil/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 78fca26f2497c4e20592792e2fe4abd5fd51e58a03a023a42a54274b7c13c419
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/mbutil/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/mbutil/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  home: https://github.com/mapbox/mbutil
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE.md
+  summary: An importer and exporter for MBTiles
+  dev_url: https://github.com/mapbox/mbutil
+
+extra:
+  recipe-maintainers:
+    - rluria14
```
Comment (marcelotrevisani) [inline recipes/mbutil/meta.yaml]: Shall you add the other maintainers?

Comment (marcelotrevisani) [reply recipes/mbutil/meta.yaml]:
```suggestion
    - rluria14
    - oblute
    - benhuff
```

Comment (rluria14) [reply recipes/mbutil/meta.yaml]: changes have been made and maintainers added.

**Pull request merged**